diff -ruN xmoto-0.1.6./Makefile.am xmoto-0.1.6/Makefile.am
--- xmoto-0.1.6./Makefile.am	2005-10-07 13:51:32.000000000 +0200
+++ xmoto-0.1.6/Makefile.am	2005-10-07 13:52:33.415174500 +0200
@@ -1,6 +1,7 @@
 INCLUDES = -Isrc
 CXXFLAGS = -w $(OPTFLAGS) -DGAMEDATADIR=\"$(datadir)/@PACKAGE@\"
 CFLAGS = -w $(OPTFLAGS) -DGAMEDATADIR=\"$(datadir)/@PACKAGE@\"
+LIBS = -lode `pkg-config lua50 --libs` @LIBS@
 SUBDIRS = bin
 
 image_SOURCES = src/image/tim.cpp src/image/tim_io_stdio.cpp \
@@ -120,7 +120,7 @@
 	src/GUIFrame.cpp src/GUIList.cpp src/GUIStatic.cpp src/GUITabView.cpp \
 	src/MotoGame.cpp src/MotoGameL.cpp src/MPhysics.cpp src/PlayerData.cpp \
 	src/Renderer.cpp src/RendererInit.cpp src/Replay.cpp src/Sound.cpp \
-	src/UserConfig.cpp $(ode_SOURCES) $(lua_SOURCES) \
+	src/UserConfig.cpp \
 	src/BSP.h \
 	src/Game.h \
 	src/GameText.h \
diff -ruN xmoto-0.1.6./src/VCommon.h xmoto-0.1.6/src/VCommon.h
--- xmoto-0.1.6./src/VCommon.h	2005-10-07 13:51:32.000000000 +0200
+++ xmoto-0.1.6/src/VCommon.h	2005-10-07 13:52:12.105842750 +0200
@@ -59,12 +59,12 @@
 #include <math.h>
 
 extern "C" {
-  #include "lua/lua.h"
-  #include "lua/lauxlib.h"
-  #include "lua/lualib.h"
+  #include <lua50/lua.h>
+  #include <lua50/lauxlib.h>
+  #include <lua50/lualib.h>
 };
 
-#include "ode/ode.h"
+#include <ode/ode.h>
 
 /* Track memory? */
 #if defined(TRACK_MEMORY)
