diff -burN xmoto-0.2.6.orig/configure.in xmoto-0.2.6/configure.in
--- xmoto-0.2.6.orig/configure.in	2007-03-03 16:00:57.000000000 +0100
+++ xmoto-0.2.6/configure.in	2007-03-04 23:55:45.000000000 +0100
@@ -15,6 +15,7 @@
 dnl libz before png!
 AC_CHECK_LIB(z, uncompress, [], [AC_MSG_ERROR(zlib required)])
 
+LIBS="$LIBS -lSDL -lGL"
 AC_CHECK_LIB(jpeg, jpeg_CreateCompress, [], [AC_MSG_ERROR(libjpeg required)])
 AC_CHECK_LIB(png, png_read_image, [], [AC_MSG_ERROR(libpng required)])
 
@@ -42,11 +43,6 @@
 )
 AC_SUBST([USE_OPENGL])
 
-if test "$USE_OPENGL" = "1"
-then
-  AX_CHECK_GL()
-fi
-
 dnl Put SDL_gfx
 AC_ARG_WITH([renderer-sdlGfx],
   [AS_HELP_STRING([--with-renderer-sdlGfx=0|1],
